script.js

```diff
@@ -0,0 +1,162 @@
+    title:
+      'Bovine Bum Vet at Ipsum Dolor Sit University',
+    work: 'The TV personality who used to be the resident vet for the annual Ken Bicton memorial pig run.',
+    photo: 'resources/images/speaker01.png',
+  },
+  {
+    name: 'Tusk Henderson',
+    title:
+      'Celebrity adventurer lorem sit since 1523',
+    work: 'The first person in history to drink his own urine on all 7 continents, who faced controversy due to being accused of bringing a cow on his adventures.',
+    photo: 'resources/images/speaker02.png',
+  },
+];
+
+for (let k = 0; k < speakersTop.length; k += 1) {
+  document.getElementById('speakers-top').innerHTML += `
+  <li class="speaker-card">
+        <span class="speaker-icon"
+          ><img src="${speakersTop[k].photo}" alt="user-icon" width="50" height="50"
+        /></span>
+        <div class="speaker-info">
+          <h3 class="speaker-name">${speakersTop[k].name}</h3>
+          <br />
+          <p class="speaker-title">${speakersTop[k].title}</p>
+          <br />
+          <p class="speaker-work">${speakersTop[k].work}</p>
+        </div>
+      </li>`;
+}
+
+const speakers = [
+  {
+    name: 'Lucy Bonham',
+    title:
+      'President of Amet Beef Council dolor of America',
+    work: 'Took over from Violet Crabtree to organize the Beef Council Annual Dinner and Barn Dance, bringing a "new look" to the traditional event.',
+    photo: 'resources/images/speaker03.png',
+  },
+  {
+    name: 'Phillip Mushroom',
+    title:
+      'International TV Actor and life coach at Ipsum Institute',
+    work: 'Son of a butcher, known for TV dramas, playing detective Beef in Beef Justice and starring role in Bankside the movie.',
+    photo: 'resources/images/speaker04.jpg',
+  },
+  {
+    name: 'Roy Gluck Jr.',
+    title:
+      ' CEO of Burgers Barrel and professional whistler',
+    work: 'Kansas-based hamburgers restaurant of the fast food nature, started by Gluck Jr.`s grandfather to be the number #8 burger chain the American Midwest.',
+    photo: 'resources/images/speaker05.jpg',
+  },
+  {
+    name: 'Michael "Frank" Franklinson',
+    title:
+      'Canadian entrepreneur and inventor of the cheesemeat',
+    work: 'Travelled to the UK to sell his company Franklinson`s new product, which he claims is the "next big thing".',
+    photo: 'resources/images/speaker06.png',
+  },
+];
+
+for (let j = 0; j < speakers.length; j += 1) {
+  document.getElementById('speakers-dynamic').innerHTML += `
+     <li class="speaker-card">
+        <span class="speaker-icon"
+          ><img src="${speakers[j].photo}" alt="user-icon" width="50"
+        /></span>
+        <div class="speaker-info">
+          <h3 class="speaker-name">${speakers[j].name}</h3>
+          <br />
+          <p class="speaker-title">${speakers[j].title}</p>
+          <br />
+          <p class="speaker-work">${speakers[j].work}</p>
+        </div>
+      </li>`;
+}
+
+const moreBttn = document.getElementById('more-bttn');
```

Comment:
- Great job implementing this button :clap: 
![Screenshot from 2021-11-19 20-02-48](https://user-images.githubusercontent.com/62705947/142677320-78b4e0b8-e72d-4013-af6d-b63ac8b82e87.png)